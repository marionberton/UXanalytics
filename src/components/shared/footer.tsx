import React from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';

import logowhite from '../../assets/logo-white.svg';

export const Footer: React.FC = () => {
  return (
    <footer className="footer bg-electricblue">
      <div className="container mt-24">
        <div className="flex justify-between lg:items-center lg:flex-row flex-col py-8">
          <div className="flex flex-row items-center my-2">
            <div>
              <img src={logowhite} alt="" />
            </div>
            <div>
              <p className="text-white pl-4 text-sm md:text-lg">
                Â© UX Analytic. All rights reserved.
              </p>
            </div>
          </div>
          <div className="flex flex-row items-center align-middle my-2">
            <div className="inline-block pr-4 py-2 mr-2 ">
              <span role="img">ðŸ¤“</span>
              <a
                className="text-white pl-4 text-base md:text-lg"
                href="/contact"
              >
                contact us
              </a>
            </div>
            <div className="inline-block px-4 py-2 m-2">
              <FontAwesomeIcon
                icon={['fab', 'twitter']}
                color="white"
                size="2x"
              />
            </div>
            <div className="inline-block px-4 py-2 m-2">
              <FontAwesomeIcon
                icon={['fab', 'medium']}
                color="white"
                size="2x"
              />
            </div>
            <div className="inline-block px-4 py-2 m-2">
              <FontAwesomeIcon
                icon={['fab', 'instagram']}
                color="white"
                size="2x"
              />
            </div>
          </div>
        </div>
      </div>
    </footer>
  );
};
