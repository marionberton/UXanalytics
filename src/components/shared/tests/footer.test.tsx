import React from 'react';
import { mount } from 'enzyme';

import { Footer } from '../footer';

describe('<Footer />', () => {
  it('renders the component', () => {
    const component = mount(<Footer />);

    expect(component.render()).toMatchSnapshot();
  });
});
