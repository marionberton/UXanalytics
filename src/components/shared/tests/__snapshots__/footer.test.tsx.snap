// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the component 1`] = `
<footer
  class="footer bg-electricblue"
>
  <div
    class="container mt-24"
  >
    <div
      class="flex justify-between lg:items-center lg:flex-row sm:flex-col py-16"
    >
      <div
        class="flex flex-row items-center"
      >
        <div>
          <img
            alt=""
            src="logo-white.svg"
          />
        </div>
        <div>
          <p
            class="text-white pl-4"
          >
            Â© UX Analytic. All rights reserved.
          </p>
        </div>
      </div>
      <div
        class="flex flex-row items-center"
      >
        <div>
          <img
            alt=""
            src="twitter.svg"
          />
        </div>
        <div>
          <a
            class="text-white pl-4"
            href="/contact"
          >
            contact us
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
